.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.ref_name }}'
+  cancel-in-progress: true
+
+jobs:
+  ci:
+    name: CI
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-13]
+
+    runs-on: ${{matrix.os}}
+    container: ${{ matrix.os == 'ubuntu-latest' && 'ghcr.io/f3d-app/f3d-superbuild-ci' || null }}
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        path: 'source'
+        fetch-depth: 0
+
+    - name: Checkout HelloWorld
+      uses: actions/checkout@v4
+      with:
+        repository: cristianadam/HelloWorld
```

Comment:
we do not have control over this repo, can we use something else?

Reply:
we can use anything that has no dependency, do you have a suggestion ?

This repository is maintained and has releases, maybe I could point a specific release to checkout instead ?

Reply:
I'm just concerned about the repository being closed.  
But yes, at least point to a tag.